crates/common/src/provider.rs

```diff
@@ -280,6 +277,28 @@ where
     provider.estimate_eip1559_fees(None).await.wrap_err("Failed fetch EIP1559 fees")
 }
 
+#[cfg(not(windows))]
+fn resolve_path(path: &Path) -> Result<PathBuf, ()> {
+    if path.is_absolute() {
+        Ok(path.to_path_buf())
+    } else {
+        Ok(env::current_dir()
+            .map(|current_dir| current_dir.join(path))
+            .expect("Current directory should exist"))
+    }
+}
+
+#[cfg(windows)]
+fn resolve_path(path: &Path) -> Result<PathBuf, ()> {
+    let path_str = path.to_str().expect("Path should be a valid string");
+
+    if path_str.starts_with(r"\\.\pipe\") {
```

Comment:
is this always the case?

Reply:
Yes. IPC on Windows don't have regular files like on Unix. 

https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea